### DISCUSSION TITLE
RabbitMQ and Erlang stable version for Production environment

### DISCUSSION THREAD

**Discussion by vaibhavjoshi888** [rabbitmq/rabbitmq-server#4060] [Other] [answered]
Hi Team,

I want to know which will be stable version of RabbitMQ and Erlang of quorum queues for Production environment
so it will run without any issue 

Regards
Vaibhav Joshi

**Comment by michaelklishin** [comment]
Thank you for your time.

Team RabbitMQ uses GitHub issues for specific actionable items engineers can work on. GitHub issues are not used for questions, investigations, root cause analysis, discussions of potential issues, etc (as defined by this team).

We get at least a dozen of questions through various venues every single day, often light on details.
At that rate GitHub issues can very quickly turn into a something impossible to navigate and make sense of even for our team. Because GitHub is a tool our team uses heavily nearly every day, the signal/noise ratio of issues is something we care about a lot.

Please post this to [rabbitmq-users](https://groups.google.com/forum/#!forum/rabbitmq-users).

Thank you.

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment, marked as answer]
The latest versions there are. There is no right answer but a version from a year ago is usually the wrong one.
